bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/NetatmoBridgeHandler.java

```diff
@@ -168,6 +168,10 @@ private void scheduleTokenInitAndRefresh() {
                             return;
                     }
                 }
+            } catch (Exception e) {
```

Comment:
Try to minimize the scope of the catch to just the runtime exceptions since any checked exceptions added in the future changes might unintentionally get handled the same.
```suggestion
            } catch (RuntimeException e) {
```

Reply:
After reading about RuntimeException vs Exception, I agree with you.